tsconfig.tsbuildinfo

```diff
@@ -0,0 +1,2328 @@
+{
```

Comment:
is this supposed to be committed?

Reply:
No, it was already .gitignored 😆
Removed ✅